pajbot/modules/basic/permaban.py

```diff
@@ -66,6 +91,12 @@ def unpermaban_command(bot, source, message, **rest):
 
             AdminLogManager.add_entry("Permaban remove", source, log_msg)
 
+            if self.settings["unban_from_chat"] is True:
+                bot.unban(user)
+
+            if self.settings["enable_send_timeout"] is True and self.settings["unban_from_chat"] is True:
+                bot.timeout(user, 1, self.settings["timeout_reason"].format(source=source), once=True)
+
```

Comment:
However I'd still like these two blocks to be collapsed
```suggestion
            if self.settings["unban_from_chat"] is True:
                bot.unban(user)

                if self.settings["enable_send_timeout"] is True:
                    bot.timeout(user, 1, self.settings["timeout_reason"].format(source=source), once=True)

```

Reply:
Wait you marked this as resolved. Did you want me to do it?

Reply:
I didn‘t mean to, oops. Yes, if you apply this change I can approve this and merge it

Reply:
I specifically added that extra statement - what is the point of the bot trying to time someone out for 1 second if they're still banned? Unless I'm missing what is actually being changed here (due to the horribly small code block).

Reply:
The way you proposed it checks unban_from_chat twice. This is just a change to make the check for the unban_from_chat setting once and then put the check for enable_send_timeout one level deeper into that if block. 
Maybe you missed the fact that I added a level of indentation

Reply:
A timeout still won‘t be done if the unban is disabled.